### DISCUSSION TITLE
Renovate GH hosted app is not picking up the repositoties

### DISCUSSION THREAD

**Discussion by visto9259** [renovatebot/renovate#28345] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I have configured the GH hosted Renovate app in the organization LM-Commons (github.com/lm-commons). Originally, I had set it for specific repos which worked fine for 2 repos. Then I added 2 more repos via the app config in the org settings but these additional repos where not picked up by the app.  Tried to set to all repos, no changes.  The new repos have composer.json and package.json, are public repos and are not forked repos.  

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Hi, there may be a backend sync problem as you are the second person today to mention this issue - we have escalated it internally

**Reply by visto9259** [reply]
Thanks.  I have the problem since yesterday (New York time)

**Reply by visto9259** [reply]
BTW, was this the right place to report the issue?

**Reply by rarkins** [reply]
Yes, this place is best

**Comment by rarkins** [comment]
This was a confirmed bug but was resolved about 12 hours ago

**Reply by visto9259** [reply]
It's working again.  Thanks
